ui/pages/confirmations/types/confirm.ts

```diff
@@ -52,3 +52,9 @@ export type ConfirmMetamaskState = {
     signatureSecurityAlertResponses?: Record<string, SecurityAlertResponse>;
   };
 };
+
+export type LastInteractedConfirmationInfo = {
```

Comment:
Could we define this in shared and reference it in the JSDoc of the controller methods?